Bottom: a74f3e3a64b8329808636a9ab8ea6c047f662a21
Top:    a253aede4b505c3b27335fe2110b58a9b3d7a6ce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-02 19:43:05 +0000

Refresh of avoid-clearing-pins-vias-from-

---

diff --git a/src/draw.c b/src/draw.c
index 8143cea..1cacc5c 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
